source/js/buyers-guide/search/creep-utils.js

```diff
@@ -18,8 +18,24 @@ export class CreepUtils {
     const container = document.querySelector(`.product-box-list`);
     const list = [...container.querySelectorAll(`.product-box`)];
     const creepVal = (e) => parseFloat(e.dataset.creepiness);
-    list
-      .sort((a, b) => creepVal(a) - creepVal(b))
-      .forEach((p) => container.append(p));
+    const alphaSort = (e) => e.querySelector(".product-name").innerText;
+    switch (history.state?.sort) {
+      case "ALPHA":
+        list
+          .sort((a, b) => alphaSort(a).localeCompare(alphaSort(b)))
+          .forEach((p) => container.append(p));
+        break;
+      case "DESCENDING":
+        list
+          .sort((a, b) => creepVal(b) - creepVal(a))
+          .forEach((p) => container.append(p));
+        break;
+      case "ASCENDING":
+      default:
+        list
+          .sort((a, b) => creepVal(a) - creepVal(b))
+          .forEach((p) => container.append(p));
+        break;
+    }
```

Comment:
Can you add some context why this is part of a utility class rather than a dedicated module / function for the sort functionality?

Reply:
The Sort was already existed before this update but we are just expanding on the existing function.  It was not a large rewrite for the sort existing function so I do not think it really needs to be moved but I am open to refactoring in the future if more comes

Reply:
What do you think about moving this method (which also could just be a function I guess) to the `sort-creepiness.js` module? I am confused by the sort functionality being spread between all these modules.

Reply:
I think the creep-utils.js and utils.js file should be merged since the separation makes less sense to me. For me the two functions in creep-utils do not make sense why they are seperated. I will address that